src/Lists/Store.php

```diff
@@ -7,7 +7,7 @@
 /**
  * Class Store
  *
- * The URL Package uses lists like Mozilla's Public Suffix List that contains all public domain suffixes.
+ * The `Url` Package uses lists like Mozilla's Public Suffix List that contains all public domain suffixes.
```

Comment:
"The url package" refers to the package name, not the class name.

Reply:
```suggestion
 * The URL Package uses lists like Mozilla's Public Suffix List that contains all public domain suffixes.
```